sdk/workflow_run.go

```diff
@@ -288,3 +290,34 @@ func (w *WorkflowNodeRunArtifact) GetPath() string {
 	container = strings.Replace(container, "/", "-", -1)
 	return container
 }
+
+const workflowNodeRunReport = `CDS Report {{.WorkflowNodeName}}#{{.Number}}.{{.SubNumber}}
+{{range $s := .Stages}}
+* {{$s.Name}}
+{{- range $j := $s.RunJobs}}
+  * {{$j.Job.Action.Name}} {{ if eq $j.Status "Success" -}} ✔ {{ else }}{{ if eq $j.Status "Fail" -}} ✘ {{ else }}- {{ end }} {{ end }}
+{{- end}}
+{{end}}
+
+{{- if gt .Tests.TotalKO 0}}
+Unit Tests Report
+
+{{- range $ts := .Tests.TestSuites}}
+* {{ $ts.Name }}
+{{range $tc := $ts.TestCases}}
+  {{- if or ($tc.Errors) ($tc.Failures) }}  * {{ $tc.Name }} ✘ {{- end}}
+{{end}}
+{{- end}}
+{{- end}}
+`
+
+func (w WorkflowNodeRun) Report() (string, error) {
```

Comment:
receiver name w should be consistent with previous receiver name nr for WorkflowNodeRun